python_terraform/__init__.py

```diff
@@ -76,6 +78,7 @@ def __init__(self, working_dir=None,
             if terraform_bin_path else 'terraform'
         self.var_file = var_file
         self.temp_var_files = VariableFiles()
+        self.workspace = workspace
```

Comment:
It looks to me like the `workspace` constructor arg is never used here.
Maybe best to remove the arg, and allow usage via the `set_workspace` method?

Reply:
Good call, thanks!  I had planned to do something different and never cleaned this up.

Reply:
What I had intended to do, and will probably add back in at a late date, was for init of the Terraform class you'd be able to pass in a workspace you desired this instance of Terraform to work against, it would then validate it was in that workspace and if not set, but I needed to just ge this working for a project at work, forked, added support, and didn't flesh it out, will do in a future PR.  I have a Jira created on my board at work to go back and fix this.

Reply:
I would avoid leaving any unused code for future work(clean code).
let's remove it here and add them back in your future PR

Reply:
Resolved this.  Cleaned up comment in __init__ also that referenced workspace since it's not used currently.